react-accordion-component/my-app/src/Accordion.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+export default function Accordion({ topics }) {
+  const [isActive, setIsActive] = useState(false);
```

Comment:
Instead of setting if no topic is selected with a Boolean of true/false, I would set to `null`